Sources/JSON.swift

```diff
@@ -24,7 +24,7 @@ import Foundation
 
 public struct JSON {
     
-    private var object: AnyObject?
+    private(set) var object: AnyObject?
```

Comment:
Since we now get `rawValue` from `RawRepresentable`, there isn't really any value in making `object` public. `object` is an internal implementation detail, so I'd rather keep it private, in case the internal implementation of `JSON` were ever to change. Plus, since `rawValue` offers the same functionality, exposing `object` would increase the scope of the public API, without any benefit.